etc/nginx/snippets/rewrites.conf

```diff
@@ -215,3 +215,27 @@ location ^~ /sdk-api/couchbase-transactions-java/ { rewrite ^/sdk-api/(couchbase
 location ^~ /sdk-api/couchbase-transactions-cxx/ { rewrite ^/sdk-api/(couchbase-transactions-cxx)/(.*)$ /sdk-api/$1-$current_version_txns_cxx_api/$2 last; }
 location ^~ /sdk-api/couchbase-transactions-dotnet/ { rewrite ^/sdk-api/(couchbase-transactions-dotnet)/(.*)$ /sdk-api/$1-$current_version_txns_dotnet_api/$2 last; }
 # location ^~ /sdk-api/couchbase-agent-catalog/ { rewrite ^/sdk-api/(couchbase-agent-catalog)/(.*)$ /sdk-api/$1-$current_version_sdk_python_ai_api/$2 last; }
+
+# bad admin/admin/ links that went variously to developer.couchbase.com or 200 of stale content
+
+location ^~ /admin/admin/ {
+    # first rewrite pages that have no known candidate to the intro page
+    rewrite ^/admin/admin/enterprise-edition.html           /admin/admin/Couchbase-intro.html;
+    rewrite ^/admin/admin/Install/upgrade-xdcr.html         /admin/admin/Couchbase-intro.html;
+    rewrite ^/admin/admin/Misc/Trbl-wrongdocs.html          /admin/admin/Couchbase-intro.html;
+    rewrite ^/admin/admin/REST/design-docs-rest-api.html    /admin/admin/Couchbase-intro.html;
```

Comment:
Might be more useful to redirect to https://docs-archive.couchbase.com/docs-3x/admin/REST/rest-views-intro.html